src/ralph/networks/admin.py

```diff
@@ -47,17 +47,59 @@ class Meta:
         exclude = ('parent',)
 
 
+def ip_address_base_object_link(obj):
+    if not obj.base_object:
+        return '&ndash;'
+    return '<a href="{}" target="_blank">{}</a>'.format(
+        reverse('admin:view_on_site', args=(
+            obj.base_object.content_type_id,
+            obj.base_object_id
+        )),
+        obj.base_object._str_with_type,
+    )
+
+
+class LinkedObjectTable(TableWithUrl):
+
+    def linked_object(self, item):
+        return ip_address_base_object_link(item)
+
+    linked_object.title = _('Linked object')
+
+
+class NetworkRalphChangeList(RalphChangeList):
+    def get_queryset(self, request):
+        """
+        Check if it has been used any filter,
+        if yes change mptt_indent_field value in model_admin to '0'
+        so as not to generate a tree list
+
+        Args:
+            request: Django Request object
+
+        Returns:
+            Django queryset
+        """
+        queryset = super().get_queryset(request)
+        filter_params = self.get_filters_params()
+        if filter_params:
```

Comment:
although this code is pretty ugly, we don't have any idea how to handle it better right now. We'll back to this in the next week.